src/content/docs/tutorial-app-performance/guide-to-monitoring-core-web-vitals.mdx

```diff
@@ -110,33 +133,9 @@ Each web vital has the following thresholds to help you track your app's perform
 
 **Good** web vitals can help your organization save money and increase SEO. **Poor** or **Needs improvement** can reflect negative user perceptions of your organization, which could result in dissatisfied customers or lost business opportunities.
 
-Want to learn more about how core web vitals are collected? Check out our
-[self-paced core web vitals training course](https://learn.newrelic.com/self-paced-core-web-vitals/). It's free!
-
-## Monitor core web vitals with New Relic [#monitor]
-
-Now that you're familiar with core web vitals, it's time to monitor them with New Relic.
-
-If you've already monitoring your browser app with New Relic, go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Browser > (select an app) > Monitor > Web vitals**. 
+## Drill down into specific pages [#drill-down]
```

Comment:
Drill down seems a little awkward. May just be me though.

Reply:
Yeah, agreed. Gonna keep it for now bc it was the original title / idk what other verb to use here that i havent used already